waddrmgr: add flags field on address row

tuxcanfly opened the pull request:
Refs #254 
WIP: Working on fetching the field from db, adding tests

jrick commented:
This looks like a pretty small and reasonable change, and shouldn't be difficult to merge either.  I'll review this soon so we can land it.

tuxcanfly commented:
Thanks. Yeah, this partially addresses the comments in #254. Still need to add the functionality to mark addresses as expired, but I think that could be done in a different PR.
